packages/design-system-react-native/src/temp-components/Spinner/Spinner.types.ts

```diff
@@ -0,0 +1,31 @@
+import type { ViewProps } from 'react-native';
+
+import type { IconColor, IconProps } from '../../components/Icon';
+import type { TextProps } from '../../components/Text/Text.types';
+
+/**
+ * Spinner component props.
+ */
+export type SpinnerProps = {
+  /**
+   * Optional prop that sets the color of the spinner icon using predefined theme colors
+   * @default IconColor.IconDefault
+   */
+  color?: IconColor;
```

Comment:
Can we continue to follow the enum naming convention of `ComponentName + PropName` for all components? For example, `Spinner` (component name) + `Color` (prop name) = `SpinnerColor`. To adhere to DRY principles, we can export `IconColor` as `SpinnerColor` from [index.ts](https://github.com/MetaMask/metamask-design-system/pull/355/files#diff-ba6856a8f8af1d27db4dbba2b96f2808fc0662c37a669de831b2b85ef55f021e)

```
import type { SpinnerColor } from '.'
```

```suggestion
  color?: SpinnerColor;
```